app/uk/gov/hmrc/securemessage/controllers/SecureMessageRenderer.scala

```diff
@@ -53,4 +55,34 @@ class SecureMessageRenderer @Inject() (
       case r => InternalServerError
     }
   }
+
+  def getContentBy(id: String, msgType: String): Action[AnyContent] = Action.async { implicit request =>
+    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
+
+    def createMsg(replyType: RenderType.ReplyType): Future[Result] =
```

Comment:
Is it actually creating message or rendering the message? Better to have right name

Reply:
updated now, thanks